src/Mermaid/Mermaid.tsx

```diff
@@ -69,10 +81,13 @@ export const MermaidAddon = (properties: MermaidProps) => {
     if (initialized) {
       return;
     }
-    const config = selectConfig(theme.palette.type, properties);
+    const config: MermaidConfig = selectConfig(theme.palette.type, properties);
     if ( properties.iconLoaders ) {
       mermaid.registerIconPacks(properties.iconLoaders);
     }
+    if (config?.layout) {
```

Comment:
I think it would be better to not assume ELK (or another layout loader) here and instead just keep this generic.

```suggestion
    if ( properties.layoutLoaders ) {
       mermaid.registerLayoutLoaders(properties.layoutLoaders);
    }
```

This way people could pass in any loader they would like to use (and this addon doesn't need to bundle all of them)/

Reply:
> I think it would be better to not assume ELK (or another layout loader) here and instead just keep this generic.
> 
> This way people could pass in any loader they would like to use (and this addon doesn't need to bundle all of them).

@johanneswuerbach interesting idea. So it would be on the implementors responsibility to include whatever renderer they'd like in their backstage app. Then they'd pass the dependency in on the new `properties.layoutLoaders`?

Reply:
And then all notion of `@mermaid-js/elk-layouts` would be removed from this plugin.

Reply:
Correct 👍